oeo-imports/uo/uo-extracted.owl

```diff
@@ -142,7 +157,7 @@
     </owl:Axiom>
     <owl:Axiom>
         <owl:annotatedSource rdf:resource="http://purl.obolibrary.org/obo/UO_0000000"/>
-        <owl:annotatedProperty rdf:resource="http://www.w3.org/2000/01/rdf-schema#comment"/>
+        <owl:annotatedProperty rdf:resource="http://purl.obolibrary.org/obo/#IAO_0000115"/>
```

Comment:
I tried to open this link in a browser and it failed. Changing several parts of the link also doesn't fix it. Ontobee is currently down, which complicates proposing a fix.

Reply:
UO uses "rdfs:comment" as annotation property for definitions. We want to change this to "IAO_0000115" (definition), to be conform to OEO. This is done for all imported classes.

Reply:
```suggestion
        <owl:annotatedProperty rdf:resource="http://purl.obolibrary.org/obo/IAO_0000115"/>
```
The problem is the `#` character in the URL. Without that, the URI resolves to: https://ontobee.org/ontology/IAO?iri=http://purl.obolibrary.org/obo/IAO_0000115